### ISSUE TITLE
Prerelease mode + commit: true: pre.json not committed

### ISSUE THREAD

**Issue by bencergazda** [atlassian/changesets#592]
## Affected Packages

@changesets/cli

## Problem

When prerelease mode is enabled, the changesets get collected in pre.json. However, if `commit` is set to true, 

## Proposed solution

I think that pre.json should be committed at the same time with the created changeset file int the `add` command

**Comment by bencergazda** [comment]
Are you open to a PR to fix this?

**Comment by Andarist** [comment]
> Are you open to review a PR for this?

Absolutely! Sorry for the late reply - I see that you have already created a PR for this. I'm at the moment going through all of the issues but once I'm done with them I'm going to switch to cleaning up PRs. In other words - I should get to your PR in some days.

**Comment by controversial** [comment]
@Andarist any updates on reviewing the open PR for this?
